astro/src/content/docs/extend/examples/multi-application-dashboard.mdx

```diff
@@ -0,0 +1,166 @@
+section: extend
+subcategory: examples
+---
+
+import Aside from 'src/components/Aside.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import IconButton from 'src/components/icon/Icon.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+
+## Overview
+
+If you're using FusionAuth to manage many applications, you might want to provide your users with a central list of applications, so they know what's available. For example, Google does this for its apps.
+
+![Google app selector](/img/docs/extend/examples/multi-application-dashboard/googleGrid.png)
+
+In this guide, you'll learn to make a similar page — a dashboard linking to all the applications in a FusionAuth tenant.
+
+In the language of authentication, FusionAuth is an identity provider (IdP) and your applications and websites are the service providers (SPs). One way to authenticate a user from a central dashboard is to use an identity-provider-initiated login. In other words, the dashboard will log the user in, and then redirect them to the selected app. In short, a service provider application will receive a login that it didn't initiate itself. This has security complexities and dangers.
+
+This guide will demonstrate a simpler and safer way to authenticate users from a central dashboard by making each application link in the dashboard point to the application's login page. From that point onward, authentication follows the standard OAuth Authorization Code Flow.
+
+The next section of this guide will show you how to make a dashboard for two existing FusionAuth applications in the same tenant. All this requires is customizing the theme for the index landing page of the FusionAuth site. The guide will first show you how to install FusionAuth and make two simple web application web servers. If you already have a FusionAuth installation with existing applications, you can skip ahead to the section that creates the dashboard.
+
+If you want to follow along with the full guide, you will need Docker installed.
+
+## Download The Example Repository And Run FusionAuth
+
+Use `git clone` to clone the repository at https://github.com/fusionauth/fusionauth-example-multiapp-dashboard, or download and unzip it. 
+
+Open a terminal in the directory containing the repository files.
+
+Run the command below to start FusionAuth.
+
+  ```sh
+  docker compose up
+  ```
+
+<Aside type="note">
+  If you have completed any FusionAuth tutorials before, you might need to first delete any existing FusionAuth containers with the same name and the database volumes. Run the command below to do that.
+
+  ```sh
+  docker ps rm fa faDb; docker compose down -v
+  ```
+</Aside>
+
+Leave FusionAuth running. 
+
+In a new terminal, run the commands below to start a web server for the Changebank app, which uses FusionAuth for authentication. 
+
+  ```sh
+  cd bankApp
+  docker run --platform=linux/amd64 --rm -v ".:/app" -w "/app" node:23-alpine3.19 sh -c  "npm install"
```

Comment:
This won't work on an ARM based chip, I'm guessing. What have you done in the past? can you create a docker compose file for these applications that is cross platform?

Reply:
it should usually 'just work' through emulation I think (we've tested this on x64 and M1 machines), but yes in general we struggle a bit with figuring out how to do this nicely as we often see little niggles on different platforms with Docker. 

That image is available on all platforms, so maybe we should just add a note at this point advising the reader to match their platform if they have issues?